bin/detail/toolchain_table.py

```diff
@@ -406,12 +406,36 @@ def verify(self):
           arch='x86',
           vs_version='16'
       ),
+      Toolchain(
+          'vs-16-2019-cxx14',
+          'Visual Studio 16 2019',
+          arch='x86',
+          vs_version='16'
+      ),
+      Toolchain(
+          'vs-16-2019-cxx17',
+          'Visual Studio 16 2019',
+          arch='x86',
+          vs_version='16'
+      ),
       Toolchain(
           'vs-16-2019-win64',
           'Visual Studio 16 2019',
           arch='amd64',
           vs_version='16'
       ),
+      Toolchain(
```

Comment:
do you want to add ninja support as well?

Reply:
I know nothing about ninja, so not in this PR.